callstats-java-sdk/src/main/java/io/callstats/sdk/messages/AuthorizeRequestMessage.java

```diff
@@ -0,0 +1,46 @@
+package io.callstats.sdk.messages;
+
+/**
+*
+* @author Karthik Budigere
+*/
+public class AuthorizeRequestMessage {
```

Comment:
Protocol specifies generic header. Perhaps it'd be better to make a "generic header" class and inherit from this class.